backend/src/routes/doctorOrders.js

```diff
@@ -177,6 +177,15 @@ router.patch('/api/updateRx/:id/pickedUp', async (req, res) => {
       }
     });
     console.log('Sent RxFill to EHR and received status from EHR', status.data);
+
+    const remsAdminStatus = await axios.post('http://rems-administrator:8090/ncpdp/script', rxFill, {
```

Comment:
I see my other comment was marked as outdated, but rems-administrator will only work on docker. That is why I am unable to successfully test this. Can you please make the host/ip configurable?